.gitignore

```diff
@@ -175,3 +175,6 @@ napari/resources/icons/_themes/
 tools/perfmon/*/traces-*.json
 
 github_cache.sqlite
+
+.gitignore
+/tests
```

Comment:
Was this on purpose? Seems wrong...

Reply:
Oh that's a mistake, I forgot the changes I made to .gitignore to add another custom tests folder